libs/dataset-explorer/src/lib/TableView/TableView.styles.ts

```diff
@@ -28,6 +27,10 @@ export const tableViewStyles: () => IProcessedStyleSet<ITableViewStyles> =
         paddingTop: 6,
         width: 36
       },
```

Comment:
Please confirm if reflow scenario is working for newly added table/view.

Reply:
done